go/vt/vttablet/tabletserver/schema/engine.go

```diff
@@ -64,6 +64,11 @@ type Engine struct {
 	notifierMu  sync.Mutex
 	notifiers   map[string]notifier
 
+	// SkipMetaCheck skips the metadata about the database and table information
+	SkipMetaCheck bool
```

Comment:
@sougou , is it ok if we add this flag here?

Reply:
Sounds good for now. I see that you're just using it to covert timestamp to gtid. There may be a simpler way to do that without having to use vstream, but we can use this method until we figure that out.